code/pizzabot.js

```diff
@@ -1,7 +1,140 @@
+        return vegPic;
+    } else if (orderName.toLowerCase() === pepperoni.toLowerCase()) {
+        return pepPic;
+    } else {
+        return hawPic;
+    }
+};
+
+//--------------------------------------------------------------------------//
+
+//Function which is connected to the first form in html inside the onsubmit() attribute of the form. When the user inputs the username and the pizza they want and presses the submit button, if true (i.e. if pizza name matches what's on the menu), it runs the code and also shows the next prompt of how many pizzas they want. If that is not true then the they get the wrongPizza message asking them to order a pizza from the menu.
+const enteredPizzaName = (event) => {
+    event.preventDefault();//Stops the page from submitting information to the backend.
+    let orderName = document.getElementById("enterPizza").value;
+    let correctPizzaName = validateOrderName(orderName);
+
+    if (correctPizzaName === true) {
+        document.getElementById("orderDifferentPizza").style.display= "none";
+        document.getElementById("howMany").innerHTML = `&#129302 And how many ${orderName}'s would you like to order?`;
+        document.getElementById("formTwo").style.display = "flex";
+        //The second form (where the user will input how many pizzas they want) will be show if true as we make that element in html to display flex. Before this time it's defined as display: none is css.
+    }
+    else {
+        document.getElementById("wrongPizza").innerHTML = "&#129302 Sorry that pizza isn't on the menu. Please order a pizza from the menu";
+        document.getElementById("orderDifferentPizza").style.display= "flex";
+    }
+};
+
+//Function that is connected to the second form in html inside the onsubmit() attribute in the form. When user inputs the amount of pizzas they want and  press the submit button, the code inside the function is run. Which is the final order message clarifying all the details of their order and an image of the pizza they've ordered.   
+const enteredPizzaAmount = (event) => {
+    event.preventDefault(); //Stops the page from submitting information to the backend.
+    let orderAmount = document.getElementById("enterAmount").value; //Taking the value (the amount) the user has inputted in the DOM for the amount of pizzas they want to order and put this inside a function to then use to calculate the cooking time below. 
+    let correctOrderAmount = calculateCookingTime(orderAmount);
+    let totalPrice = calculateTotalCost(orderAmount, pizzaPrice);
+    let orderName = document.getElementById("enterPizza").value;
+    let pizzaImages = pizzaImage(orderName);
+    document.getElementById("formTwo").style.display = "flex";
+
+    //Defined variable and if statement which helps to change if the pizza if plural or not in the final order statement.
+    let plural = "'s";
+    if (orderAmount === "1") {
+        plural = "";
+    }
```

Comment:
Yes yes yes yes. And more yes. 💯

Reply:
Thanks! We got a bit of help with this so can't really take all the credit :) It is a good thing to know tho!